source/stylesheets/modules/_year_filter.scss

```diff
@@ -5,11 +5,14 @@
 .year-filter {
 
   > .headline {
-    @include screenreaders-only;
+    @include filter-headline;
+    float: right;
```

Comment:
it's required because the filter headlines are now `display: inline-block;` so that I can control how wide their bottom border is. `text-align` would work for the text, but the border would be 100%